### ISSUE TITLE
DLND-Week2(3)

### ISSUE THREAD

**Issue by chenghong-lin-nu** [Hclmaster/blog#5]
### Graphs
- 神经网络是一个由数学函数所组成的图。（例如线性组合和激活函数）
![image](https://user-images.githubusercontent.com/16880879/36151540-8d17f25c-1102-11e8-9006-a96a1d5db89d.png)

### Forward Propagation
- By propagating values from the first layer (the input layer) through all the mathematical functions represented by each node, the network outputs a value. This process is called a **forward pass**.

### Gradient Descent
- gradient: 相当于是我们应该往哪里去push ball
- how much force should be applied to the push. -> This is known as **learning rate**.  -> 它决定了网络学的有多快。
- 定义learning rate有点像在猜数字游戏，但是一般来说在0.1 to 0.0001 会work well。（The range 0.001 to 0.0001 is popular, as 0.1 and 0.01 are sometimes too large.）
